Sources/SwiftUIBackports/iOS/PresentationDetents/Detents.swift

```diff
@@ -249,9 +254,9 @@ private extension Backport.Representable {
 
                 UIView.animate(withDuration: 0.25) {
                     if let undimmed = largestUndimmed {
-                        controller.presentingViewController.view.tintAdjustmentMode = (selection?.wrappedValue ?? .large) >= undimmed ? .automatic : .normal
+                        controller.presentingViewController.view?.tintAdjustmentMode = (selection?.wrappedValue ?? .large) >= undimmed ? .automatic : .normal
                     } else {
-                        controller.presentingViewController.view.tintAdjustmentMode = .automatic
+                        controller.presentingViewController.view?.tintAdjustmentMode = .automatic
```

Comment:
Can you verify this? Just confused how this was compiling before without `?` – I don’t think its necessary?

Reply:
Without unwrapping the view (or rather with the default force-unwrapping), the code seems to be crashing when closing the sheet. `willTransition` is probably also called then, but `view` is already set to `nil`

Reply:
ah interesting ok thanks